### ISSUE TITLE
Coding Style

### ISSUE THREAD

**Issue by pjabardo** [tunelipt/VisualEFM.jl#2]
The code follows a coding style common in R codes or Microsoft (camelCase). Wouldn't it be better to use more Julian coding style (https://docs.julialang.org/en/v1/manual/style-guide/) ?

**Comment by gborelli89** [comment]
Yes, you are right. I'll try to modify code style as soon as I can. Feel free to provide any additional changes in code style or in the code itself to make it more efficient.
